### PR TITLE
Load template translations params

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: assets/static/template.go [hunk 1]
```diff
@@ -38,21 +38,23 @@ func (t *Template) Translations() []assets.TemplateTranslation {
...
-	Channel_       *assets.ChannelReference `json:"channel"         validate:"required"`
-	Content_       string                   `json:"content"         validate:"required"`
-	Locale_        i18n.Locale              `json:"locale"          validate:"required"`
-	Namespace_     string                   `json:"namespace"`
-	VariableCount_ int                      `json:"variable_count"`
+	Channel_       *assets.ChannelReference   `json:"channel"         validate:"required"`
+	Content_       string                     `json:"content"         validate:"required"`
+	Locale_        i18n.Locale                `json:"locale"          validate:"required"`
+	Namespace_     string                     `json:"namespace"`
+	VariableCount_ int                        `json:"variable_count"`
```
Comment (rowanseymour) [inline assets/static/template.go:45]: Can't we get rid of variable count? Isn't it just `len(params["body"])` ?

Comment (norkans7) [reply assets/static/template.go:45]: Yes those are the same, I have not seen where that variable is used actually, I guess probably with the flow_editor, so we can cleanup that once we are sure it is not relied on

Comment (rowanseymour) [reply assets/static/template.go:45]: I mean get rid of it here if the engine doesn't need it

Comment (norkans7) [reply assets/static/template.go:45]: Okay

**Pull request merged**